build.sh

```diff
@@ -65,7 +65,7 @@ function configure {
 }
 
 function mak {
-    pushd build && make VERBOSE=1 -j$NPROC
+    pushd build && make -j$NPROC
```

Comment:
Can pushd fail? If so, do you want to popd?

Reply:
@RossBrunton  this is because the length of the log would be more that 10000 lines in travis and then travis cannot shows the logs and any result should be seen in a raw log. That is why we change that to be able to see the travis result.

Reply:
We have safeguards in place for when pushd fails... But if it does, then nothing will be on the stack when popd runs.

(This is nitpicky, and I don't see it as a blocker)

Reply:
But that's a different issue, so shouldn't be fixed here.